pkg/lib/operation/dbt/generate/sources/operation_test.go

```diff
@@ -0,0 +1,207 @@
+package sources
+
+import (
+	"testing"
+
+	"github.com/keboola/go-client/pkg/storageapi"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTablesByBucketsMap(t *testing.T) {
```

Comment:
Kompletní E2E test na tohle zatím nenapíšu, tak jsem se to snažil pokrýt unit testama. Výsledek z reálnýho projektu pak vypadá takto:
![CleanShot 2022-09-26 at 17 04 18](https://user-images.githubusercontent.com/215660/192312300-0250800a-0d01-4c13-b35e-755d2434f2b1.png)